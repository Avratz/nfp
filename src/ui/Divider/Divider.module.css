.divider {
  all: unset;
  display: block;
  background-color: var(--neutral-3);
}

.horizontal {
  width: 100%;
  height: 1px;
}
.vertical {
  width: 1px;
  height: 100%;
}
