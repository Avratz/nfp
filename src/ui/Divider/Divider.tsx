import { cx } from 'class-variance-authority'
import styles from './Divider.module.css'

export function Divider({ className = '' }: { className?: string }) {
  return <hr className={cx(styles.divider, className)} />
}
