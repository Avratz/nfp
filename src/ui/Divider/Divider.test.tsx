import { render } from '@testing-library/react'
import { Divider } from './Divider'

describe('Divider', () => {
  it('renders default', () => {
    const { container } = render(<Divider />)
    expect(container).toMatchSnapshot()
  })

  it('renders vertical', () => {
    const { container } = render(<Divider vertical />)
    expect(container).toMatchSnapshot()
  })
})
