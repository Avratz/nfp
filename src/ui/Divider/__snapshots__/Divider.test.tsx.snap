// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider renders default 1`] = `
<div>
  <hr
    class="divider horizontal"
  />
</div>
`;

exports[`Divider renders vertical 1`] = `
<div>
  <hr
    class="divider vertical"
  />
</div>
`;
